SDL/PixelTriplet.cu

```diff
@@ -360,52 +368,53 @@ __device__ bool SDL::passPT3RPhiChiSquaredInwardsCuts(struct modules& modulesInG
     const int layer1 = modulesInGPU.layers[lowerModuleIndex1] + 6 * (modulesInGPU.subdets[lowerModuleIndex1] == SDL::Endcap) + 5 * (modulesInGPU.subdets[lowerModuleIndex1] == SDL::Endcap and modulesInGPU.moduleType[lowerModuleIndex1] == SDL::TwoS);
     const int layer2 = modulesInGPU.layers[lowerModuleIndex2] + 6 * (modulesInGPU.subdets[lowerModuleIndex2] == SDL::Endcap) + 5 * (modulesInGPU.subdets[lowerModuleIndex2] == SDL::Endcap and modulesInGPU.moduleType[lowerModuleIndex2] == SDL::TwoS);
     const int layer3 = modulesInGPU.layers[lowerModuleIndex3] + 6 * (modulesInGPU.subdets[lowerModuleIndex3] == SDL::Endcap) + 5 * (modulesInGPU.subdets[lowerModuleIndex3] == SDL::Endcap and modulesInGPU.moduleType[lowerModuleIndex3] == SDL::TwoS);
-    
-    if(layer1 == 7 and layer2 == 8 and layer3 == 9)
+ 
+    if(layer1 == 7 and layer2 == 8 and layer3 == 9) // endcap layer 1,2,3, ps
     {
         return chiSquared < 22016.8055f;
     }
-    else if(layer1 == 7 and layer2 == 8 and layer3 == 14)
+    else if(layer1 == 7 and layer2 == 8 and layer3 == 14) // endcap layer 1,2,3 layer3->2s
     {
         return chiSquared < 935179.56807f;
     }
-    else if(layer1 == 8 and layer2 == 9 and layer3 == 10)
+    else if(layer1 == 8 and layer2 == 9 and layer3 == 10) // endcap layer 2,3,4
     {
         return chiSquared < 29064.12959f;
     }
-    else if(layer1 == 8 and layer2 == 9 and layer3 == 15)
+    else if(layer1 == 8 and layer2 == 9 and layer3 == 15) // endcap layer 2,3,4, layer3->2s      
+    // why don't we have endcap 3,4,5 for 3 ps modules? that's 9,10,11
```

Comment:
Did we answer that in a meeting? Should we discuss it and remove this comment?

Reply:
yes, that's a leftover question when I first saw the code and has been answered